server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/api/posttask/PostProjectAnalysisTasksExecutor.java

```diff
@@ -147,13 +153,16 @@ private QualityGateImpl createQualityGate(QualityGateHolder qualityGateHolder) {
     private final CeTask ceTask;
     private final Project project;
     private final Date date;
+    private final ScannerContext scannerContext;
     @CheckForNull
     private final QualityGate qualityGate;
 
-    private ProjectAnalysis(CeTask ceTask, Project project, Date date, @Nullable QualityGate qualityGate) {
+    private ProjectAnalysis(CeTask ceTask, Project project, Date date,
+      ScannerContext scannerContext, @Nullable QualityGate qualityGate) {
       this.ceTask = requireNonNull(ceTask, "ceTask can not be null");
       this.project = requireNonNull(project, "project can not be null");
       this.date = requireNonNull(date, "date can not be null");
+      this.scannerContext = requireNonNull(scannerContext, "scannerContext can not be null");
```

Comment:
FYI, any of the three is acceptable: can not (less favored), cannot, can't.



---
*G. Ann CAMPBELL* | SonarSource
Product Owner
http://sonarsource.com

On Tue, Aug 9, 2016 at 10:22 AM, Sébastien Lesaint <notifications@github.com
> wrote:

> In server/sonar-server/src/main/java/org/sonar/server/computation/task/
> projectanalysis/api/posttask/PostProjectAnalysisTasksExecutor.java
> <https://github.com/SonarSource/sonarqube/pull/1136#discussion_r74067665>:
>
> >        this.ceTask = requireNonNull(ceTask, "ceTask can not be null");
> >        this.project = requireNonNull(project, "project can not be null");
> >        this.date = requireNonNull(date, "date can not be null");
> > +      this.scannerContext = requireNonNull(scannerContext, "scannerContext can not be null");
>
> @simonbrandhof <https://github.com/simonbrandhof> , since I'm trying
> myself to stop doing it, I'm passing to you this comment initially from
> @teryk <https://github.com/teryk> and backed up by @ganncamp
> <https://github.com/ganncamp> : this should be scannerContext can't be
> null in American English
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SonarSource/sonarqube/pull/1136/files/9e543d777132764faded6d2581afc53d305a4ec9#r74067665>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQK0ym_F3GVlNheJgZb9yiYv_CITuVyks5qeI09gaJpZM4JePsc>
> .
>